src/lib.rs

```diff
@@ -604,3 +604,61 @@ pub fn selfdestruct(address: &Address) -> ! {
         native::ethereum_selfDestruct(address.bytes.as_ptr() as *const u32);
     }
 }
+
+pub trait EwasmAPI: Send + Sync {
```

Comment:
Do we need the `Send + Sync` traits here? Those are for concurrency control.

Reply:
We don't, that's just a leftover.